android_10/src/org/ros/android/MasterChooser.java

```diff
@@ -348,6 +372,50 @@ protected boolean isQRCodeReaderInstalled(Intent intent) {
   }
 
   /**
+   * Stores the recent Master URI to preferences. This implementation does not use
+   * {@link android.content.SharedPreferences.Editor#putStringSet(String, Set)}
+   * since it is not available in API 10.
+   * @param uri Master URI string to store.
+   */
+  private void setRecentMasterURI(String uri)  {
```

Comment:
please review formatting. For example: open should go on the same line as the control statement with one blank space (not zero, not two), etc.
Hint: it is possible to configure Android Studio to do this for you.